CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 See the [versioning documentation for how to update this changelog](./docs/contributing/versioning.md#updating-changelog).
 
+🆕 New features:
+
+- Meta title
```

Comment:
The title you have in your pull request might be more descriptive for the changelog.